verification/curator-service/ui/src/components/new-case-form-fields/LocationForm.tsx

```diff
@@ -132,7 +138,13 @@ export function PlacesAutocomplete(
                         name="unused"
                         required={props.required}
                         data-testid={props.name}
-                        label="Location"
+                        // Use the initial valuelocation name as a hint when untouched
```

Comment:
Is there not a defaultValue that can be used rather than changing the label?

Reply:
nope, tried hard but {value}, {text}, {initialValue}, {defaultValue} etc nothing worked except label.

Reply:
Hm that's a shame. The material UI TextField component has a defaultValue field: https://material-ui.com/api/text-field/ Maybe we could try using that TextField rather than the formik-material-ui one? I thought all the formik-material-ui components passed through props to the material-ui component... shame it's not working as expected.

Reply:
good idea, let me try with that field now and merge if I don't get anywhere...

Reply:
nopenopenope considering this a _good enough_ fix for now to not not lose too much time on it...

FTR got a mixed bags of errors like
```
Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in the componentWillUnmount method.
```

or `TypeError: Cannot read property 'isSubmitting' of undefined` when trying to use something else than the current TextField :(